run.py

```diff
@@ -1,90 +1,126 @@
 #!/usr/bin/env python3
 import json
-import os, os.path as op
-import subprocess as sp
+import os
+import logging
 import sys
 
-import flywheel
-from utils.custom_logger import get_custom_logger
-from utils import bids, args, results
+from gear_toolkit import gear_toolkit_context
+from gear_toolkit import bids, zip_tools
+from utils import args
+
+
+log = logging.getLogger(__name__)
 
-if __name__ == '__main__':
-    # Preamble: take care of all gear-typical activities.
-    context = flywheel.GearContext()
-    #get_custom_logger is defined in utils.py
-    context.log = get_custom_logger('[flywheel:C-PAC]')
+
+def main(context):
 
     # grab environment for gear
     with open('/tmp/gear_environ.json', 'r') as f:
         environ = json.load(f)
 
-    # This gear will use a "custom_dict" dictionary as a custom-user field 
+    # This gear will use a "custom_dict" dictionary as a custom-user field
     # on the gear context.
-    context.custom_dict ={}
+    context.custom_dict = {}
 
     context.custom_dict['environ'] = environ
 
-    try: # To download and validate BIDS 
-        # A routine to validate on a "shadow" structure is in development
-        context.custom_dict['bids_dir'] = op.join(context.work_dir,'bids')
-        bids.download(context)
-        bids.run_validation(context)
+    try:
+        # To download and validate BIDS
+        context.download_session_bids(target_dir=context.work_dir/'bids')
```

Comment:
I switched from using `download_session_bids()` to `download_project_bids()` because the project version also downloads "dataset_description.json" and the session version didn't.  Has that changed?  I guess what I'm asking is, does the validator accept it?  I assume it must.  I guess I need to look at this again.